munimap/templates/munimap/app/index.html

```diff
@@ -145,9 +145,11 @@
 
     angular.module('munimapDraw')
     .value('StyleSchema', {$ style_schema | tojson $})
-    .value('StyleSchemaFormOptions', {$ style_schema_form_options | tojson $})
+    {# TODO: causes TypeError: '<' not supported between instances of 'int' and 'str' #}
+    {# .value('StyleSchemaFormOptions', {$ style_schema_form_options | tojson $}) #}
```

Comment:
apparently the problem is because in Python 3 `tojson` fails to sort keys when it processes json.dumps(dict, sort_keys=True). 

`json.dumps()` default value for sort_keys is False. However, in Flask, the default app configs states the value for JSON_SORT_KEYS to be True. ([Check here](https://flask.palletsprojects.com/en/0.12.x/api/?highlight=json_sort_keys).

This problem is fixed by adding `JSON_SORT_KEYS = False` to the app config.